runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1340,6 +1368,21 @@ gcParseReconfigurableCommandLine(J9JavaVM* vm, J9VMInitArgs* args)
 		}
 		extensions->softMx = softmx;
 	}
+	if (-1 != FIND_ARG_IN_ARGS(args, EXACT_MEMORY_MATCH, "XXgc:fvtest_RAMSize=", NULL)) {
+		double testRAMSize = 0;
```

Comment:
I would select different name to avoid confusion with `extensions->testRAMSize`, something like `testRAMSizePercentage`

Reply:
I think the same applicable with option name `fvtest_RAMSize=` as well. It is not an absolute size, but percentage.